source/unified-test-format/unified-test-format.rst

```diff
@@ -2779,7 +2782,13 @@ events for the following:
 
 - Any commands containing sensitive information (per the
   `Command Monitoring <../command-monitoring/command-monitoring.rst#security>`__
-  spec).
+  spec) unless
+  `observeSensitiveCommands <entity_client_observeSensitiveCommands_>`_ is true.
+  Note that drivers will redact commands and replies for sensitive commands. For
+  ``hello`` and legacy hello, which are conditionally sensistive based on the
+  presence of a ``speculativeAuthenticate`` field, the test runner may need to
+  infer that the events are sensitive based on whether or not the command and
+  reply documents are redacted (i.e. empty documents).
```

Comment:
@prashantmital: I just added this to capture the point made in my last PR comment. Let me know if this helps, as I expect other drivers may run into similar issues.

Reply:
Yup, this is perfect... There is a distinction between command redaction and events being issued/captured which is obviously implied by the original language but made much more explicit with the addition of this new text. LGTM!